src/main/java/duke/ui/Ui.java

```diff
@@ -0,0 +1,18 @@
+package duke.ui;
+
+public class Ui {
```

Comment:
Maybe you can consider bringing the user input/output here since `UI`  is supposed to "deals with interactions with the user". I personally made all my print methods and `Scanner` in this class so that the I/O stuff is isolated here.